packages/arb-rpc-node/batcher/sequencerBatcher.go

```diff
@@ -697,14 +705,14 @@ func (b *SequencerBatcher) publishBatch(ctx context.Context, dontPublishBlockNum
 				if b.config.Node.Sequencer.RewriteSequencerAddress {
 					b.reorgToNewSequencerAddress(ctx, prevMsgCount)
```

Comment:
Not from this PR, but should we be checking for returned error from reorgToNewSequencerAddress?